pom.xml

```diff
@@ -150,6 +150,11 @@
 			<artifactId>bigdataviewer-vistools</artifactId>
 			<version>${bigdataviewer-vistools.version}</version>
 		</dependency>
+        <dependency>
+            <groupId>mobie</groupId>
+            <artifactId>bigdataviewer-imageloader</artifactId>
+            <version>1.0.1-SNAPSHOT</version>
```

Comment:
Should we wait for merging this until https://github.com/mobie/bigdataviewer-imageloader/pull/16 is merged and released? Then we can use the artifact from scijava.maven and don't need the snapshot dependency here.